File: BEE1010/main.c
#include <stdio.h>
#include <stdlib.h>

int main () {

    int codigo1;
    int peca1;
    float valor_peca1;

    int codigo2;
    int peca2;
    float valor_peca2;

    scanf("%d %d %f", &codigo1, &peca1, &valor_peca1);
    scanf("%d %d %f", &codigo2, &peca2, &valor_peca2);

    printf("VALOR A PAGAR: R$ %.2f\n",(peca1*valor_peca1)+(peca2*valor_peca2));
	
	return 0;
}

File: BEE1010/src/Main.java
import java.util.Locale;
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Locale.setDefault(Locale.US);
        Scanner ler = new Scanner(System.in);

        int cod1 = ler.nextInt();
        int peca1 = ler.nextInt();
        double valorPeca1 = ler.nextDouble();

        int cod2 = ler.nextInt();
        int peca2 = ler.nextInt();
        double valorPeca2 = ler.nextDouble();

        double total = peca1 * valorPeca1 + peca2 * valorPeca2;

        System.out.printf("VALOR A PAGAR: R$ %.2f%n", total);

        ler.close();
    }
}